src/index.js

```diff
@@ -48,7 +48,14 @@ router.post(
 			extraHTTPHeaders: body.headers,
 		});
 		const page = await context.newPage();
-		await page.goto(body.url);
+		await page.goto(body.url, {
+			waitUntil: "domcontentloaded",
+		});
+		await page.waitForFunction(() => {
+			// eslint-disable-next-line
+			return document.fonts.ready;
```

Comment:
where is document coming from? 🤔

Reply:
Playwright magic, that'swhy eslint ignore is also needed.

Did manual QA and it worked.

Here is link to docs: https://playwright.dev/docs/api/class-page#page-wait-for-function

```
const selector = '.foo';
await page.waitForFunction(selector => !!document.querySelector(selector), selector);
```
> Snippet from docs